### DISCUSSION TITLE
Simple Bug - Uploader Error - Request Entity Too Large At 1.1Mb not 20Mb

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#4980] [Help] [answered]
Hello! Just getting one issue / bug and I'd like some clarification please.

When I try to upload images, the uploader will not take an image larger than ~1.05Mb, yet the attachment docs say it should take 20Mb uploads. It will not take 6x 200Kb files, or one 1.1Mb file. T
![Screenshot 2022-03-16 at 15 56 34](https://user-images.githubusercontent.com/93920520/159283161-5c70b1e8-6586-4880-a30e-7bcdeb65f0df.png)
![Screenshot 2022-03-16 at 15 56 37](https://user-images.githubusercontent.com/93920520/159283184-abb35c97-7e4e-4c5f-b280-c7fa6de5a34d.png)
his seems really bizarre as, again, the docs says 20Mbs. Please let me know what could be the issue.

Screen shots and bug reports aren't very significant, just trying to upload a single file, it responds with "request entity too large".

**Comment by shogunpurple** [comment, marked as answer]
Hi @CosmasSC128 - this seems to be a limit on our NGINX configuration. I can update this to allow for 20MB. I'll push an update ASAP.

**Reply by CosmasSC128** [reply]
Thank you so much.
